Removed VSCode config from codelab_rebuild.yaml

SorinCR opened the pull request:
VSCode config was messing up the launch on VSCode for different devices:
No supported devices found with name or id matching 'iPhone'

![image](https://github.com/flutter/codelabs/assets/48986428/17d1a3c8-4221-4452-baf6-f0d4dd120535)

## Pre-launch Checklist

- [x] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [X] I signed the [CLA].
- [X] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.

domesticmouse commented on firebase-get-to-know-flutter/codelab_rebuild.yaml:
```diff
@@ -54,55 +54,55 @@ steps:
-              }
-            ]
-          }
+      # - name: VSCode config
```
Don't comment out, delete outright. You'll also need to re-run the `codelab_rebuild` tooling to make it take effect.

SorinCR replied:
Hey, I made the change.
How can I run codelab_rebuild?

domesticmouse replied:
See https://github.com/flutter/codelabs/blob/main/tooling/codelab_rebuild/README.md

domesticmouse commented:
I'm thinking that instead of removing the VSCode launch config, we could just remove the named devices. That should eliminate the issue without removing the ease of getting started with a debug target.

SorinCR commented:
I have removed the named devices, codelab_rebuild still has some issues.
I'll look into them later

` > codelab_rebuild .\firebase-get-to-know-flutter\codelab_rebuild.yaml
INFO: 2023-08-23 09:55:14.916115: Get to know Firebase with Flutter
INFO: 2023-08-23 09:55:14.936708: step_02
INFO: 2023-08-23 09:55:14.936708: Remove generated code
INFO: 2023-08-23 09:55:15.041574: Create project
INFO: 2023-08-23 09:55:17.357415: Creating project gtk_flutter...
INFO: 2023-08-23 09:55:20.749772: Resolving dependencies in gtk_flutter...
INFO: 2023-08-23 09:55:20.755306:
INFO: 2023-08-23 09:55:22.674635: Got dependencies in gtk_flutter.
INFO: 2023-08-23 09:55:22.675634:
INFO: 2023-08-23 09:55:23.200768: Wrote 108 files.
INFO: 2023-08-23 09:55:23.201280:
All done!
INFO: 2023-08-23 09:55:23.207324: You can find general documentation for Flutter at: https://docs.flutter.dev/
Detailed API documentation is available at: https://api.flutter.dev/
If you prefer video documentation, consider: https://www.youtube.com/c/flutterdev

In order to run your application, type:

  $ cd gtk_flutter
  $ flutter run

Your application code is in gtk_flutter\lib\main.dart.
INFO: 2023-08-23 09:55:23.465691: Strip DEVELOPMENT_TEAM
INFO: 2023-08-23 09:55:23.478803: Configure analysis_options.yaml
INFO: 2023-08-23 09:55:23.479803: Add dependencies
INFO: 2023-08-23 09:55:28.147398: Resolving dependencies...
INFO: 2023-08-23 09:55:28.280560:   collection 1.17.1 (1.18.0 available)
+ crypto 3.0.3
+ ffi 2.1.0
+ flutter_web_plugins 0.0.0 from sdk flutter
+ go_router 10.1.0
INFO: 2023-08-23 09:55:28.280560: + google_fonts 5.1.0
+ http 1.1.0
+ http_parser 4.0.2
+ logging 1.2.0
  matcher 0.12.15 (0.12.16 available)
  material_color_utilities 0.2.0 (0.8.0 available)
+ path_provider 2.1.0
+ path_provider_android 2.1.0
+ path_provider_foundation 2.3.0
+ path_provider_linux 2.2.0
+ path_provider_platform_interface 2.1.0
+ path_provider_windows 2.2.0
+ platform 3.1.1
+ plugin_platform_interface 2.1.5
  source_span 1.9.1 (1.10.0 available)
  stack_trace 1.11.0 (1.11.1 available)
INFO: 2023-08-23 09:55:28.281626:
  stream_channel 2.1.1 (2.1.2 available)
  test_api 0.5.1 (0.6.1 available)
+ typed_data 1.3.2
+ win32 5.0.6
+ xdg_directories 1.0.2
INFO: 2023-08-23 09:55:28.284648: Changed 19 dependencies!
INFO: 2023-08-23 09:55:28.285648:
INFO: 2023-08-23 09:55:29.373576: Patch pubspec.yaml
Unhandled exception:
ProcessException: The system cannot find the file specified.

  Command: patch -u c:\users\sorin\onedrive\desktop\proiecte\codelabs\firebase-get-to-know-flutter\gtk_flutter\pubspec.yaml
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401:33)
#1      Process.start (dart:io-patch/process_patch.dart:38:20)
#2      _buildBlueprintStep (package:codelab_rebuild/src/rebuild_blueprint.dart:271:31)
#3      _buildBlueprintStep (package:codelab_rebuild/src/rebuild_blueprint.dart:50:13)
<asynchronous suspension>
#4      rebuildFromBlueprint (package:codelab_rebuild/src/rebuild_blueprint.dart:25:5)
<asynchronous suspension>
Building package executable...
Built codelab_rebuild:codelab_rebuild.
INFO: 2023-08-23 09:55:31.495229: Get to know Firebase with Flutter
INFO: 2023-08-23 09:55:31.507789: step_02
INFO: 2023-08-23 09:55:31.507789: Remove generated code
WARNING: 2023-08-23 09:55:31.521695: Invalid rmdir for 'c:\users\sorin\onedrive\desktop\proiecte\codelabs\firebase-get-to-know-flutter\step_02': Remove generated code
Unhandled exception:
PathNotFoundException: Deletion failed, path = 'c:\users\sorin\onedrive\desktop\proiecte\codelabs\firebase-get-to-know-flutter\step_02' (OS Error: The system cannot find the file specified.
, errno = 2)
#0      _Directory._deleteSync (dart:io/directory_impl.dart:193:7)
#1      FileSystemEntity.deleteSync (dart:io/file_system_entity.dart:424:7)
#2      _rmdir (package:codelab_rebuild/src/rebuild_blueprint.dart:387:18)
#3      _buildBlueprintStep (package:codelab_rebuild/src/rebuild_blueprint.dart:78:7)
#4      _buildBlueprintStep (package:codelab_rebuild/src/rebuild_blueprint.dart:50:13)
#5      rebuildFromBlueprint (package:codelab_rebuild/src/rebuild_blueprint.dart:25:11)
#6      Blueprint.rebuild (package:codelab_rebuild/src/blueprint.dart:64:42)
#7      main (file:///C:/Users/sorin/OneDrive/Desktop/proiecte/codelabs-main/tooling/codelab_rebuild/bin/codelab_rebuild.dart:23:13)
#8      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:294:33)
#9      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:189:12)`

domesticmouse commented:
I have yet to find a good cross platform way to do a diff/patch. So yeah, windows doesn't work. Sorry about that

domesticmouse commented:
Thankw @SorinCR!

Royes09 commented:
thx @SorinCR <3
